src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/GetTaskByIdControllerTest.java

```diff
@@ -363,4 +363,127 @@ public void should_replace_additional_properties_in_configuration_dmn_and_return
         common.cleanUpTask(taskId);
     }
 
+    @Test
+    public void should_return_a_200_with_task_id_in_description_when_creating_standalone_task() {
```

Comment:
I think both these tests more suitable in PostTaskInitiateByIdControllerTest. Am I right thinking that discription is created in task initiation process and the get task returning whatever the task info in cft_task db? If that's the case, there is nothing new in get operation here than the other tests in the same class. Also, it is better if you can test the discription in any existing test rather than adding a new one (only if you can).

Reply:
I think you got a point, we are touching the initiation endpoint, so will move the test PostTaskInitiateByIdControllerTest